lib/Doctrine/ORM/Query/Expr.php

```diff
@@ -47,14 +47,14 @@ public function andX($x = null)
      *     // (u.type = ?1) OR (u.role = ?2)
      *     $q->where($q->expr()->orX('u.type = ?1', 'u.role = ?2'));
      *
-     * @param mixed $x Optional clause. Defaults to null, but requires
+     * @param Expr\Comparison|Expr\Func|Expr\Orx|string ...$clauses Optional clauses. Defaults to an empty array, but requires
```

Comment:
Please also add `Andx` to the list, see https://github.com/doctrine/orm/pull/8098/files

Reply:
Done! Added to orX method too.